lib/sensors/GPS.h

```diff
@@ -0,0 +1,339 @@
+//UBlox SAM-M8Q-0-10 GPS Sensor'
+
```

Comment:
can you also add a link to the datasheet?

Reply:
Added in newest commit